src/DependencyInjection/CompilerPass/GenerateAdminServicesCompilerPass.php

```diff
@@ -100,7 +100,7 @@ public function process(ContainerBuilder $container)
                 $contentItemDefinition = $container->getDefinition($contentItemServiceId);
 
                 if (in_array($contentItemDefinition->getArgument(1), $matrix->getTypes())) {
-                    $pageDef->addMethodCall('addChild', [new Reference($contentItemServiceId), 'contentItems']);
+                    $pageDef->addMethodCall('addChild', [new Reference($contentItemServiceId)]);
```

Comment:
Nog iets dat ik tegenkwam in een project (en bij het fixen van sonata-project/SonataAdminBundle#7705) dat dat tweede argument dus de property in de child moet zijn waar de parent op gekoppeld is. Dus hier niet `'contentItems'` (= children), maar `'page'` (= de parent).